### ISSUE TITLE
Mock up FAQ page

### ISSUE THREAD

**Issue by melchoyce** [GSA-TTS/FAC#767]
Breakout issue of #377, #457.

This issue covers the design of the future FAC FAQs page.

### Acceptance criteria (We'll know we're done when...)

- [ ] Users can locate answers to common questions they have about using the FAC.
  
### Tasks

- [x] Design
- [ ] Peer review
- [ ] Write implementation notes

### Definition of Done

- [ ] Screens in Figma are ready to be handed off to front end
- [x] ~All text has a contrast ratio of 4.5:1 with the background~ (Built into USWDS)
- [x] Interaction are documented if they deviate from USWDS components
- [x] The needs of someone navigating the site with the use of a screenreader or keyboard has been considered and documented 
- [x] ~There is an option for multilingual support~ (We've established multilingual support in the header)
- [x] There is supporting documentation for alt text, labels for links (anchor + target), error text in forms, tab order, focus order, and what assistive technologies should announce (if applicable)

**Comment by melchoyce** [comment]
Still pending final copy per #457, but here's a design start for the FAQs page:

![image](https://user-images.githubusercontent.com/2846578/216672441-80a18341-5406-4b4e-9e88-f847f4a5ed5f.png)
[Figma](https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=5725%3A33297&t=Wuq6OT3Qgwxucjds-4)

The left nav displays the top-level FAQ categories.

**Comment by melchoyce** [comment]
Chatted about this during our design backlog chat today. Putting it in our backlog to revisit and update once we have a chance to review the copy for these pages.

**Comment by jadudm** [comment]
Closing as done/not planned. We have a team working on content, and it is informed by SME input and users.
